ROOTFS/media/rpi/sage-utils/dhcp-pxe/tftp/cmdline.txt

```diff
@@ -1 +1 @@
-console=serial0,115200 console=tty1 root=/dev/nfs nfsroot=10.31.81.1:/media/rpi/sage-utils/dhcp-pxe/nfs,vers=3 ro ip=dhcp rootwait elevator=deadline init=/bin/ro-root.sh
+console=serial0,115200 console=tty1 root=/dev/nfs nfsroot=10.31.81.1:/media/rpi/sage-utils/dhcp-pxe/nfs,vers=3 ro ip=dhcp rootwait elevator=deadline init=/bin/ro-root.sh cgroup_memory=1 cgroup_enable=memory
```

Comment:
Do you still need these cgroup changes for Ubuntu?  

Are you sure this `cmdline.txt` file is still being used?  I sorta assumed the boot procedure for ubuntu would be very different then Raspbian ?  maybe not....

Also, the `init=/bin/ro-root.sh' part?  i assumed this was Raspbian only and didn't apply to Ubuntu.  This is what got me thinking if this cmdline.txt file is even being used anymore.

You can check on your running system by looking at `/proc/cmdline` or something

Reply:
The cgroup's were still a necessary change, was getting error messages from k3s about them